plugins/cockroachdb_logs.yaml

```diff
@@ -0,0 +1,260 @@
+            layout: '%y%m%d %H:%M:%S.%f'
+            location: {{ .timezone }}
+          severity:
+            parse_from: attributes.Type
+    {{ end }} # End of sql schema logs
+    
+    {{ if .enable_telemetry_log }}
+    filelog/telemetry_logs:
+      include:
+        {{ range $fp := .telemetry_log_path }}
+        - '{{ $fp }}' 
+        {{ end }} 
+      multiline:
+        line_start_pattern: '[IWEF]\d{6}\s\d{2}:\d{2}:\d{2}\.\d{6}\s'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'cockroach.telemetry'
+      operators:
+        - type: regex_parser
+          regex: '^(?P<Type>[IWEF])(?P<timestamp>\d{6}\s+\d{2}:\d{2}:\d{2}\.\d{6})\s+(?P<GoID>\d+)\s*(?:(?P<Channel>\d+)@)?(?P<Location>\S+)\s+(?:)⋮?\s+(?P<Node>\[[^\]]+\])\s+(?:(?P<Counter>\d+\s+)?)(?P<Message>.*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%y%m%d %H:%M:%S.%f'
+            location: {{ .timezone }}
+          severity:
+            parse_from: attributes.Type
+    {{ end }} # End of telemetry logs
+
+    {{ if .enable_kv_distribution_log }}
+    filelog/kv_distribution_logs:
+      include:
+        {{ range $fp := .kv_distribution_log_path }}
+         - '{{ $fp }}'
+        {{ end }} 
+      multiline:
+        line_start_pattern: '[IWEF]\d{6}\s\d{2}:\d{2}:\d{2}\.\d{6}\s'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'cockroach.kv.distribution'
+      operators:
+        - type: regex_parser
+          regex: '^(?P<Type>[IWEF])(?P<timestamp>\d{6}\s+\d{2}:\d{2}:\d{2}\.\d{6})\s+(?P<GoID>\d+)\s*(?:(?P<Channel>\d+)@)?(?P<Location>\S+)\s+(?:)⋮?\s+(?P<Node>\[[^\]]+\])\s+(?:(?P<Counter>\d+\s+)?)(?P<Message>.*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%y%m%d %H:%M:%S.%f'
+            location: {{ .timezone }}
+          severity:
+            parse_from: attributes.Type
+    {{ end }} # End of kv distribution logs
+    
+    {{ if .enable_pebble_log }}
+    filelog/pebble_logs:
+      include:
+        {{ range $fp := .pebble_log_path }}
+         - '{{ $fp }}'
+        {{ end }} 
+      multiline:
+        line_start_pattern: '[IWEF]\d{6}\s\d{2}:\d{2}:\d{2}\.\d{6}\s'
+      start_at: {{ .start_at }}
+      attributes:
+        log_type: 'cockroach.pebble'
+      operators:
+        - type: regex_parser
+          regex: '^(?P<Type>[IWEF])(?P<timestamp>\d{6}\s+\d{2}:\d{2}:\d{2}\.\d{6})\s+(?P<GoID>\d+)\s*(?:(?P<Channel>\d+)@)?(?P<Location>\S+)\s+(?:)⋮?\s+(?P<Node>\[[^\]]+\])\s+(?:(?P<Counter>\d+\s+)?)(?P<Message>.*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%y%m%d %H:%M:%S.%f'
+            location: {{ .timezone }}
+          severity:
+            parse_from: attributes.Type
+    {{ end }} # End of pebble logs.
+
+  service:
+    pipelines:
+      logs:
+        receivers:
+          {{ if .enable_health_log }}
+          - filelog/health_logs
```

Comment:
Should all of these named be prefixed with cockroachdb? IE `filelog/cockroachdb_health_logs`